bin/main.rb

```diff
@@ -0,0 +1,28 @@
+puts "Welcome to tic-tac-to game"
+
+puts "Player 1 please enter your name"
+player1_name = gets.chomp
+puts "#{player1_name} please enter symbol you want to use"
+player1_symbol = gets.chomp
+
+puts "Player 2 please enter your name"
+player2_name = gets.chomp
+puts "#{player2_name} please enter symbol you want to use"
+player1_symbol = gets.chomp
+
+game_over = false
+player_turn = 1
+while game_over == false
+  if player_turn.odd?
```

Comment:
You should be displaying your board by now (even if you will change it later) So that the players can see what the available mover are.
You should show it before and after a player's move.